view/loadView.js

```diff
@@ -108,8 +110,14 @@ function samlAuthentication(userProfile, done) {
       });
     }
 
-    // profile already attached - default scope applied on find
-    return done(null, user);
+    if (user.fullName) {
+      // profile already attached - default scope applied on find
+      return done(null, user);
+    }
+
+    user.update({ // user.fullName doesn't exist, update user
+      fullName: userFullName,
+    }).then(() => done(null, user));
```

Comment:
Does the "user" which is passed on in the `done(null, user)` actually include the new fullName attribute? Test?

Reply:
@iamigo This test was here but I updated some wording to make this clearer https://github.com/salesforce/refocus/pull/789/files#diff-3036175005cd1bfda68f7b60355102bdR129 .. The test uses the user that was created in the beforeHook (without a fullName), and then checks that the fullName is there after